lib/hsa/hsaRfiUpdateBadInputs.cpp

```diff
@@ -67,14 +93,14 @@ hsa_signal_t hsaRfiUpdateBadInputs::execute(int gpu_frame_id, hsa_signal_t prece
     set_rfi_num_bad_inputs(_network_buf, _network_buf_execute_id, bad_inputs_correlator.size());
```

Comment:
There is a subtitle, but important thing here.  `bad_inputs_correlator` should be removed from this object since we no longer update it.  However that means the frames we get will have to contain the number of bad inputs and we'll have to record that so that we can continue to update the metadata here with the right value.